wsbroadcastserver/clientconnection.go

```diff
@@ -86,6 +88,13 @@ func (cc *ClientConnection) Start(parentCtx context.Context) {
 			case <-ctx.Done():
 				return
 			case data := <-cc.out:
+				time.Sleep(cc.delay)
```

Comment:
can't sleep in this thread, danger of `cc.out` filling up and blocking.  Need to maintain cache of messages if not sending right away.

Reply:
The write to cc.out is non blocking and we log if we drop any messages. We could just increase the cc.out size if we're worried about dropping messages. Delivery is best-effort anyway so we're not guaranteeing to deliver.

Reply:
Thought about it a bit more and I agree with your feedback. We're incurring a fixed cost for each connection for whatever we set `MaxSendQueue` to so we don't want to increase for all connections it just to handle potential spillovers from new connections which will be relatively fewer. We can allow a variable sized buffer per-connection for handling overflows that can grow as needed.